lib/e2e-proc/src/lib.rs

```diff
@@ -9,7 +9,6 @@ mod test;
 ///
 /// ```rust,ignore
 /// #[e2e::test]
-/// async fn foo(alice: User, bob: User) -> eyre::Result<()> {
```

Comment:
👀

Reply:
I know, already reverted this change 😅

Reply:
Yeah, I was reviewing when you updated and the comment stayed. That's why I resolved the conversation.